### PR TITLE
Backup: add link to CLI section

### PR DESCRIPTION
**Pull request by c0ffeeca7**
## Type of change

- [x] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

## Summary by CodeRabbit

- **Documentation**
  - Updated backup-related documentation to include a reference to the Home Assistant Command Line instructions.

### CODE REVIEW

**Comment by frenck** [comment]
@coderabbitai review

File: source/_includes/common-tasks/backups.md [hunk 1]
```diff
@@ -40,7 +40,7 @@ A partial backup consists of any number of the above default directories and ins
 2. `ha backups restore slugname` - restores a specific backup
 3. `ha backups new --name nameofbackup` - create a backup
 
-Use `ha help` to get more information about the command line usage.
+Use `ha help` to get more information about the command line usage or refer to the documentation on the [Home Assistant Command Line](/common-tasks/os/#home-assistant-via-the-command-line).
```
[the remark this answers is not in the document]
Comment (c0ffeeca7) [reply source/_includes/common-tasks/backups.md:43]: Why did you remove drop `ha help`?

[the remark this answers is not in the document]
Comment (c0ffeeca7) [reply source/_includes/common-tasks/backups.md]: > Consider using a more formal tone for documentation.
> 
> ```diff
> - Use `ha help` to get more information about the command line usage or refer to the documentation on the [Home Assistant Command Line](/common-tasks/os/#home-assistant-via-the-command-line).
> + For additional information about command line usage, please refer to the [Home Assistant Command Line documentation](/common-tasks/os/#home-assistant-via-the-command-line).
> ```
> 
> Committable suggestion
> > ‼️ **IMPORTANT**
> > Carefully review the code before committing. Ensure that it accurately replaces the highlighted code, contains no missing lines, and has no issues with indentation.

@coderabbitai Why did you remove ha help?

[the remark this answers is not in the document]
Comment (c0ffeeca7) [reply source/_includes/common-tasks/backups.md]: > @c0ffeeca7, thank you for pointing out the omission. The initial suggestion aimed to enhance the formality of the documentation, but it inadvertently excluded the `ha help` command, which is indeed useful for providing immediate command line assistance.
> 
> Here's a revised suggestion that includes the `ha help` command while maintaining a formal tone:
> 
> ```
> For additional information about command line usage, you may use the `ha help` command or refer to the [Home Assistant Command Line documentation](/common-tasks/os/#home-assistant-via-the-command-line).
> ```
> 
> This version ensures that users have both immediate help through the CLI and detailed guidance via the documentation link.

@coderabbitai For instructional content, use the indicative mood. Do not use expressions of politeness such as 'may' or 'please'

[the remark this answers is not in the document]
Comment (c0ffeeca7) [reply source/_includes/common-tasks/backups.md]: > @c0ffeeca7, thank you for your guidance. Based on your feedback, here's the updated text that adopts a more direct and authoritative tone:
> 
> ```
> Use the `ha help` command or refer to the [Home Assistant Command Line documentation](/common-tasks/os/#home-assistant-via-the-command-line) for additional information about command line usage.
> ```
> 
> This adjustment ensures the instructional content is clear and direct, as per your specifications.

@coderabbitai The goal of an instruction should be fronted.

[the remark this answers is not in the document]
Comment (c0ffeeca7) [reply source/_includes/common-tasks/backups.md]: @coderabbitai Perfect! Can you remember these rules for future reviews?

[the remark this answers is not in the document]
Comment (c0ffeeca7) [reply source/_includes/common-tasks/backups.md]: > For instructional content in documentation, use a direct and authoritative tone. Avoid expressions of politeness such as 'may' or 'please', and ensure the goal of the instruction is fronted.

Can you apply the [Microsoft Style Guide](https://learn.microsoft.com/en-us/style-guide/welcome/)?

[the remark this answers is not in the document]
Comment (c0ffeeca7) [reply source/_includes/common-tasks/backups.md]: > @c0ffeeca7, based on your feedback, here's the rephrased instruction with the goal fronted:
> 
> ```
> For additional information about command line usage, use the `ha help` command or refer to the [Home Assistant Command Line documentation](/common-tasks/os/#home-assistant-via-the-command-line).
> ```
> 
> This version places the primary goal at the beginning of the sentence, aligning with your directive for clear and direct instructional content.

@coderabbit Provide your sentence as a suggestion

Comment (c0ffeeca7) [inline source/_includes/common-tasks/backups.md]:
```suggestion
For additional information about command line usage, use the `ha help` command or refer to the [Home Assistant Command Line documentation](/common-tasks/os/#home-assistant-via-the-command-line).
```

**Review by frenck — APPROVED** [review]
Thanks, @c0ffeeca7 👍 

../Frenck

**Pull request merged**